doc/contributing/troubleshooting.rst

```diff
@@ -27,13 +28,13 @@ See :ref:`upgrading-pypsa`.
 Consistency check on network
 ============================
 
-A consistency check can be performed using the function :py:meth:`pypsa.Network.consistency_check`.
+A consistency check can be performed using the function :py:meth:`pypsa.n.consistency_check`.
```

Comment:
If you rename just the reference the link will break. We could still rename it and keep the reference via 
```
:py:meth:`n.consistency_check <pypsa.Network.consistency_check>`
```

I have a mini helper script somewhere for those things, will check tomorrow